src/component.js

```diff
@@ -207,8 +207,10 @@ var QuillComponent = createClass({
 		if (this.quillDelta) {
 			this.editor.setContents(this.quillDelta);
 			this.editor.setSelection(this.quillSelection);		
```

Comment:
According to the quill docs, calling setSelection will also focus the editor. I think `setSelection()` should only be called, if `quillSelection` is not null/undefined.

Reply:
> According to the quill docs, calling setSelection will also focus the editor

It doesn't look like it calls focus if the selection (e.g. `quillSelection`) is null/undefined. Regardless, it's probably best not to call it anyway if it's undefined.